File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.cs
public class Solution {
    public int RemoveDuplicates(int[] nums) {
        if(nums.Length<=1)
            return nums.Length;
        if(nums.Length==2){
            if(nums[0]==nums[1])
                return 1;
            return 2; 
        }
        int index=0;
        for(int i=0;i<nums.Length-1;i++){
            if(nums[i]!=nums[i+1]){
                nums[index++]=nums[i];

            }
        }

        nums[index++]=nums[nums.Length-1];
        return index;
    }
}

File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.java
class Solution {
    public int removeDuplicates(int[] nums) {
        
        int n=nums.length;
        int prev=0;
        for(int i=0;i<n;i++){
            if(nums[i]!=nums[prev]){
                nums[++prev]=nums[i];
            }
            
        }
        return prev+1;
    }
}